src/components/OverviewComponent/Overview.js

```diff
@@ -64,127 +58,142 @@ const Overview = (props) => {
     dispatch(fetchProductStyle(props.productId));
   }, []);
 
+
+  // expands image when expand icon is clicked on a top right corner of a picture
+  const toggleCrap = () => {
```

Comment:
Might want to consider a different name for this function

Reply:
lol